### ISSUE TITLE
RFE: Allow users to choose year sanitization

### ISSUE THREAD

**Issue by abelcheung** [KristoforMaynard/music-tag#37]
I was surprised that almost half of the recent forks were there solely to circumvent or alter how year sanitization works (just before repeating the same thing myself). If it can be presented as a module setting, like:

```python
import music_tag

music_tag.YEAR_SANITIZE = True(default) | False(disable) | Callable
```

Then all those forks as well as a bunch of open issues don't need to exist (= less headache). What do you think?

**Comment by abelcheung** [comment]
Cueing @KristoforMaynard ...

**Comment by KristoforMaynard** [comment]
The feature to skip type normalization / sanitization already exists,

``` python
import music_tag as mt

f = mt.load_file('sample/440Hz-2.m4a')
f.raw['year'] = 'asdf'
yy = f.raw['year']
print(yy.value)
```

This is documented in ``Readme.md``, but if the verbiage isn't clear, then I'm open to a better description.

**Comment by abelcheung** [comment]
Thanks @KristoforMaynard, I have continued reading the other reports soon after submitting mine, and already found addition of `f.raw`. Though a bit inconvenient (no dict-like interface), but I can use `f.mfile.keys() / .items()` to make up for that, so it's not a problem.

The reason for me to retract the reports is not the above reason though. Originally I thought this is a general purpose tag manipulation library on top of mutagen, but I found myself wrong after [stumbled upon your comment](https://github.com/KristoforMaynard/music-tag/issues/1#issuecomment-629506904). With its intended goal already achieved, this library has done its job very well, and there's no need for those feature requests that does not fit the goal. Probably it's this point that needs to be made clear in readme.
